.appveyor.yml

```diff
@@ -13,25 +13,34 @@ environment:
     - SIMDJSON_BUILD_STATIC: ON
       SIMDJSON_ENABLE_THREADS: OFF
       SIMDJSON_PLATFORM: x64
+      CMAKE_EXTRA_ARGS:
     - SIMDJSON_BUILD_STATIC: ON
       SIMDJSON_ENABLE_THREADS: OFF
       SIMDJSON_PLATFORM: Win32
+      CMAKE_EXTRA_ARGS:
     - SIMDJSON_BUILD_STATIC: OFF
       SIMDJSON_ENABLE_THREADS: ON
       SIMDJSON_PLATFORM: x64
+      CMAKE_EXTRA_ARGS:
+    - SIMDJSON_BUILD_STATIC: OFF
+      SIMDJSON_ENABLE_THREADS: ON
+      SIMDJSON_PLATFORM: x64
+      CMAKE_EXTRA_ARGS: -T ClangCL
```

Comment:
I can't find this in the [cmake docs](https://cmake.org/cmake/help/v3.17/variable/CMAKE_GENERATOR_TOOLSET.html#variable:CMAKE_GENERATOR_TOOLSET), but if it works, it works :)

Reply:
Right. I tried many things. :-)